pkg/action/release_testing.go

```diff
@@ -110,17 +112,38 @@ func (r *ReleaseTesting) GetPodLogs(out io.Writer, rel *release.Release) error {
 	for _, h := range rel.Hooks {
 		for _, e := range h.Events {
 			if e == release.HookTest {
-				req := client.CoreV1().Pods(r.Namespace).GetLogs(h.Name, &v1.PodLogOptions{})
-				logReader, err := req.Stream(context.Background())
-				if err != nil {
-					return errors.Wrapf(err, "unable to get pod logs for %s", h.Name)
+				var (
+					podList *corev1.PodList
+					err     error
+				)
+				switch h.Kind {
+				case "Job":
+					podList, err = client.CoreV1().Pods(r.Namespace).List(context.Background(), metav1.ListOptions{
+						LabelSelector: labels.FormatLabels(map[string]string{"job-name": h.Name}),
+					})
+				case "Pod":
+					podList, err = client.CoreV1().Pods(r.Namespace).List(context.Background(), metav1.ListOptions{
+						FieldSelector: labels.FormatLabels(map[string]string{"metadata.name": h.Name}),
+					})
 				}
 
-				fmt.Fprintf(out, "POD LOGS: %s\n", h.Name)
-				_, err = io.Copy(out, logReader)
-				fmt.Fprintln(out)
 				if err != nil {
-					return errors.Wrapf(err, "unable to write pod logs for %s", h.Name)
+					return errors.Wrapf(err, "unable to get pod list for %s/%s", h.Kind, h.Name)
+				}
+
+				for _, pod := range podList.Items {
+					req := client.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{})
+					logReader, err := req.Stream(context.Background())
+					if err != nil {
+						return errors.Wrapf(err, "unable to get pod logs for %s/%s", h.Kind, h.Name)
+					}
+
+					fmt.Fprintf(out, "POD LOGS: %s\n", pod.Name)
+					_, err = io.Copy(out, logReader)
+					fmt.Fprintln(out)
```

Comment:
Should any extra newlines be printed after the logs for a pod, to make it clear where one pod's logs start and the next begins?

Reply:
<img width="956" alt="image" src="https://user-images.githubusercontent.com/51436614/196856499-5915bb3e-4a1c-4c84-87fb-c338c1e3d82b.png">
Does this ok?